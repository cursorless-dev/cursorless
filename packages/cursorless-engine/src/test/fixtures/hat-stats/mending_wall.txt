Mending Wall
by Robert Frost, 1914 (public domain)

Something there is that doesn’t love a wall,
That sends the frozen-ground-swell under it,
And spills the upper boulders in the sun;
And makes gaps even two can pass abreast.
The work of hunters is another thing:
I have come after them and made repair
Where they have left not one stone on a stone,
But they would have the rabbit out of hiding,
To please the yelping dogs. The gaps I mean,
No one has seen them made or heard them made,
But at spring mending-time we find them there.
I let my neighbor know beyond the hill;
And on a day we meet to walk the line
And set the wall between us once again.
We keep the wall between us as we go.
To each the boulders that have fallen to each.
And some are loaves and some so nearly balls
We have to use a spell to make them balance:
‘Stay where you are until our backs are turned!’
We wear our fingers rough with handling them.
Oh, just another kind of out-door game,
One on a side. It comes to little more:
There where it is we do not need the wall:
He is all pine and I am apple orchard.
My apple trees will never get across
And eat the cones under his pines, I tell him.
He only says, ‘Good fences make good neighbors.’
Spring is the mischief in me, and I wonder
If I could put a notion in his head:
‘Why do they make good neighbors? Isn’t it
Where there are cows? But here there are no cows.
Before I built a wall I’d ask to know
What I was walling in or walling out,
And to whom I was like to give offense.
Something there is that doesn’t love a wall,
That wants it down.’ I could say ‘Elves’ to him,
But it’s not elves exactly, and I’d rather
He said it for himself. I see him there
Bringing a stone grasped firmly by the top
In each hand, like an old-stone savage armed.
He moves in darkness as it seems to me,
Not of woods only and the shade of trees.
He will not go behind his father’s saying,
And he likes having thought of it so well
He says again, ‘Good fences make good neighbors.’
