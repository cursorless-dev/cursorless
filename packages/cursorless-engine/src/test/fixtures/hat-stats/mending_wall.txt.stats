nTokens: 479

tokenSpark:
	▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▃▁▁▃▃▃▁▃▁▁▁▃▃▁▃▃▁▃▃▃▃▃▃▃▁▃▁▃▁▃▁▃▃▁▃▃▃▃▃▃▃▁▃
	▃▃▃▃▃▃▃▃▁▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃
	▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▅▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃▃
	▅▃▅▃▅▃▃▃▃▃▃▃▃▃▃▃▃▃▃▁▃▃▃▃▃▃▃▃▃▃▅▃▁▃▃▃▃▃▃▃▃▃▃▅▃▃▃▃▅▃▃▃▃▃▃▅▅▃▃▃
	▅▃▃▅▅▃▃▃▅▃▃▃▃▃▃▃▃▃▃▃▃▅▅▃▃▅▃▃▅▃▃▃▅▃▃▃▃▃▃▃▃▃▃▃▃▅▅▃▅▅▃▃▅▃▃▅▃▃▅▅
	▅▅▃▃▃▃▃▃▃▃▃▁▅▃▅▅▃▃▅▃▃▃▅▅▅▅▃▃▃▅▃▅▃▅▃▃▃▅▃▅▅▅▃▅▅▃▃▃▃▅▅▅▅▃▅▃▃▃▅▅
	▅▅▅▃▅▃▅▅▅▅▅▅▅▅▃▅▃▃▃▃▃▅▅▅▅▅▃▅▅▃▁▅▅▅▅▅▅▅▅▅▃▃▅▅▅▅▅▅▅▅▃▃▃▅▃▅▃▅▅▃
	▅▅▃▅▃▅▅▅▃▅▃▅▅▅▅▅▅▅▃▅▃▅▅▅▃▅▅▅▅▅▅▅▅▅▅▅▃▅▅▅▃▃▅▅▅▅▅▅▃▅▅▅▅▃▅▅▅▅▅

nHats:
	mean: 100%
	std: 0%
	min: 100%
	max: 100%
	spark: 0              25               50               75             █ 100

nPenalty0:
	mean: 8%
	std: 0%
	min: 7%
	max: 8%
	spark: 0    █         25               50               75               100

nPenalty1:
	mean: 63%
	std: 1%
	min: 62%
	max: 63%
	spark: 0              25               50      ▂█       75               100

nPenalty2:
	mean: 30%
	std: 1%
	min: 29%
	max: 31%
	spark: 0              25   ██          50               75               100
