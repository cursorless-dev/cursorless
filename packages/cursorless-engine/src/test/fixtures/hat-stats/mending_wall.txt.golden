_       _
Mending Wall
[-----] [--]

_  _      _    _ _    __      _     _
by Robert Frost, 1914 (public domain)
[] [----] [---]| [--] |[----] [----]|

_         _     _   _    _   _A _    _ A   A
Something there is that doesn’t love a wall,
[-------] [---] [] [--] [---]|| [--] | [--]|

B     _    C      _  __     AA     _     A B
That sends the frozen-ground-swell under it,
[--] [---] [-] [----]|[----]|[---] [---] []|

 _  B      D   A     A        B  E   C  _
And spills the upper boulders in the sun;
[-] [----] [-] [---] [------] [] [-] [-]|

A     _   A     _   F   _   A    B      _
And makes gaps even two can pass abreast.
[-] [---] [--] [--] [-] [-] [--] [-----]|

1      A A  A       C  C       2    _
The work of hunters is another thing:
[-] [--] [] [-----] [] [-----] [---]|

D B    A    D     3    E   A    A
I have come after them and made repair
| [--] [--] [---] [--] [-] [--] [----]

B        _ C    A    A   B   D     C  F E    C
Where they have left not one stone on a stone,
[---] [--] [--] [--] [-] [-] [---] [] | [---]|

B   4    C     D    5   B      D   E  E     D
But they would have the rabbit out of hiding,
[-] [--] [---] [--] [-] [----] [-] [] [----]|

6  B       F  A       A   A  1  B    E B   E
To please the yelping dogs. The gaps I mean,
[] [----] [-] [-----] [--]| [-] [--] | [--]|

B  F   2   F     3   C    1  4      5   D   F
No one has seen them made or heard them made,
[] [-] [-] [--] [--] [--] [] [---] [--] [--]|

C   1  1      E      B F   D  A     6     A  B
But at spring mending-time we find them there.
[-] [] [----] [-----]|[--] [] [--] [--] [---]|

1 B   F  C        B    D        B  2  A
I let my neighbor know beyond the hill;
| [-] [] [------] [--] [----] [-] [--]|

2   2  3 B   E  1     3 F      C C
And on a day we meet to walk the line
[-] [] | [-] [] [--] [] [--] [-] [--]

4   2     D 1    E       B  4    5    C
And set the wall between us once again.
[-] [-] [-] [--] [-----] [] [--] [---]|

2  C      E 3    F       C  6  4  C D
We keep the wall between us as we go.
[] [--] [-] [--] [-----] [] [] [] []|

                     A
 5 F      1 1        1      A  B       6 2   E
To each the boulders that have fallen to each.
[] [--] [-] [------] [--] [--] [----] [] [--]|

 D  3     C  D       E  4    5  F      2
And some are loaves and some so nearly balls
[-] [--] [-] [----] [-] [--] [] [----] [---]

        A      A       A
5    B  2  D   1 6     3  2      3  3      A
We have to use a spell to make them balance:
[] [--] [] [-] | [---] [] [--] [--] [-----]|

A   B 6     C    D  E      F  4      E   1    _B
‘Stay where you are until our backs are turned!’
|[--] [---] [-] [-] [---] [-] [---] [-] [----]||

 4  5    2  C       F      3     1        6 F
We wear our fingers rough with handling them.
[] [--] [-] [-----] [---] [--] [------] [--]|

A
1 1 _     2      D     D  3 CC    D   2
Oh, just another kind of out-door game,
[]| [--] [-----] [--] [] [-]|[--] [--]|

       A                A
 3   4 2  4  1 5  B     4  E      3   B
One on a side. It comes to little more:
[-] [] | [--]| [] [---] [] [----] [--]|

               A  A              A
   1     2  6  1  1  D  5   6    5     F C
There where it is we do not need the wall:
[---] [---] [] [] [] [] [-] [--] [-] [--]|

A  A               A
1  2   1  C      E 3  4  D     3     2
He is all pine and I am apple orchard.
[] [] [-] [--] [-] | [] [---] [-----]|

5   E     4      2    C   E    C
My apple trees will never get across
[] [---] [---] [--] [---] [-] [----]

    A   A               A          A
  F 1   6   D     4     2   F    3 4   3    63
And eat the cones under his pines, I tell him.
[-] [-] [-] [---] [---] [-] [---]| | [--] [-]|

A
3    4    D 4 CF    E        E  1       2     4D
He only says, ‘Good fences make good neighbors.’
[] [--] [--]| |[--] [----] [--] [--] [-------]||

       A  B            A  A       B
 1     5  1      E     6  1 5   1 1    2
Spring is the mischief in me, and I wonder
[----] [] [-] [------] [] []| [-] | [----]

   B           A A      B  A
 F 2 F     2   3 1      3  4      3D
If I could put a notion in his head:
[] | [---] [-] | [----] [] [-] [--]|

                                  B   B B
E  E 4     F   F  3       4     _ 4  F2 5
‘Why do they make good neighbors? Isn’t it
|[-] [] [--] [--] [--] [-------]| [-]|| []

            A             A    B     A   A
   5     6  4   1   A 5   5    3     5   2  2   5
Where there are cows? But here there are no cows.
[---] [---] [-] [--]| [-] [--] [---] [-] [] [--]|

       B       A      C       B
6      6  5    6   5  115   1 4  2
Before I built a wall I’d ask to know
[----] | [---] | [--] ||| [-] [] [--]

A    C A           C  A
2    2 3     6     3  2        5  6 6
What I was walling in or walling out,
[--] | [-] [-----] [] [] [-----] [-]|

    B  A    C A        B
  6 5  4    4 5     3  6  6     1     6
And to whom I was like to give offense.
[-] [] [--] | [-] [--] [] [--] [-----]|

A         C     C  C    A     C      B A   A
1         1     5  2    1    23   D  1 6   1
Something there is that doesn’t love a wall,
[-------] [---] [] [--] [---]|| [--] | [--]|

C    B     C  A   A  D                   C  A  A
4    1     6  2   13 1 3       1 4  E  5 5  6  2
That wants it down.’ I could say ‘Elves’ to him,
[--] [---] [] [--]|| | [---] [-] |[---]| [] [-]|

A   D  A A                A B   DAA A
1   2 62 3     F    _     3 2   313 1
But it’s not elves exactly, and I’d rather
[-] []|| [-] [---] [-----]| [-] ||| [----]

B  A    D             A D A   B   C
1  3    4  2         32 5 4   2   6
He said it for himself. I see him there
[] [--] [] [-] [-----]| | [-] [-] [---]

A        B A                       D
2        3 5         3   4       2 1     4
Bringing a stone grasped firmly by the top
[------] | [---] [-----] [----] [] [-] [-]

D       B   A      B  A   A            B    A
6    4  3   4   4  4  3  D6       1    5    3
In each hand, like an old-stone savage armed.
[] [--] [--]| [--] [] [-]|[---] [----] [---]|

B        E           B  E  B     D  A A
4    2   1     5     6  2  1     2  2 5
He moves in darkness as it seems to me,
[] [---] [] [------] [] [] [---] [] []|

A      B          C   D   B        D    A
4    5 2        3 1   3   2      6 4    4
Not of woods only and the shade of trees.
[-] [] [---] [--] [-] [-] [---] [] [---]|

B  B    A   A  A      B   A     AB       A
5  3    5   1  3      6   1     23   4   6
He will not go behind his father’s saying,
[] [--] [-] [] [----] [-] [----]|| [----]|

C   C               D       A  E  B  B
2   1    6     3    5       4  3  4  4
And he likes having thought of it so well
[-] [] [---] [----] [-----] [] [] [] [--]

C       C    B AA           A    A    A        AA
2    5  3    1 32       5   3    3    6        54
He says again, ‘Good fences make good neighbors.’
[] [--] [---]| |[--] [----] [--] [--] [-------]||
